zerver/tests/test_messages.py

```diff
@@ -629,6 +629,34 @@ def test_applying_markdown(self):
         self.assertEqual(message.rendered_content, expected_content)
         self.assertEqual(message.rendered_content_version, bugdown.version)
 
+    @mock.patch("zerver.lib.message.bugdown.convert")
+    def test_applying_markdown_invalid_format(self, convert_mock):
+        # type: () -> None
```

Comment:
oh, thank you! I didn't see a new argument in other cases where mock was being used, so I thought it wasn't necessary. I'll know better next time!